test/test_build.py

```diff
@@ -77,10 +77,7 @@ def image_type_fixture(shared_tmpdir, build_container, request, force_aws_upload
     """
     container_ref, images, target_arch, local = parse_request_params(request)
 
-    if not local:
```

Comment:
Would you mind opening a separate PR with this fix? It seems orthogonal to the discussion in here (and useful on it's own)

Reply:
Yeah absolutely, will do that now